{
    "docstring": "\"\"\"\n    Custom result class that triggers a PDB session when an error or failure\n    occurs.\n    \"\"\"",
    "name": "PDBDebugResult",
    "namespace": "django.test.runner.PDBDebugResult",
    "module_name": "django.test.runner",
    "package_name": "",
    "methods": [
        {
            "docstring": "",
            "name": "addError",
            "namespace": "django.test.runner.PDBDebugResult.addError",
            "class_name": "django.test.runner.PDBDebugResult",
            "module_name": "",
            "package_name": ""
        },
        {
            "docstring": "",
            "name": "addFailure",
            "namespace": "django.test.runner.PDBDebugResult.addFailure",
            "class_name": "django.test.runner.PDBDebugResult",
            "module_name": "",
            "package_name": ""
        },
        {
            "docstring": "",
            "name": "addSubTest",
            "namespace": "django.test.runner.PDBDebugResult.addSubTest",
            "class_name": "django.test.runner.PDBDebugResult",
            "module_name": "",
            "package_name": ""
        },
        {
            "docstring": "",
            "name": "debug",
            "namespace": "django.test.runner.PDBDebugResult.debug",
            "class_name": "django.test.runner.PDBDebugResult",
            "module_name": "",
            "package_name": ""
        }
    ]
}